porter/v1/cluster_control_plane.proto

```diff
@@ -148,6 +148,8 @@ message ListImagesForRepositoryResponse {
 message Image {
   string repository_name = 1;
   string tag = 2;
+  string digest = 3;
+  google.protobuf.Timestamp updated_at = 4;
```

Comment:
do these fields need to be on porter app image definition as well?

Reply:
No, they are just used by the list image api call to display available images to the user.